File: Basic-Video-Chat-ConnectionService-Java/app/src/main/java/com/tokbox/sample/basicvideochatconnectionservice/deviceselector/AudioDeviceSelectionListener.java
package com.tokbox.sample.basicvideochatconnectionservice.deviceselector;

public interface AudioDeviceSelectionListener {
    /**
     * Is called when the user selects an audio device.
     * @param device The selected audio device.
     */
    void onAudioDeviceSelected(AudioDeviceSelector.AudioDevice device);
}

File: Basic-Video-Chat-ConnectionService-Kotlin/app/src/main/java/com/vonage/basic_video_chat_connectionservice/deviceselector/AudioDeviceSelectionListener.kt
package com.vonage.basic_video_chat_connectionservice.deviceselector

interface AudioDeviceSelectionListener {
    /**
     * Is called when the user selects an audio device.
     * @param device The selected audio device.
     */
    fun onAudioDeviceSelected(device: AudioDeviceSelector.AudioDevice)
}